x/incentives/keeper/gauge_test.go

```diff
@@ -225,99 +224,3 @@ func (suite *KeeperTestSuite) TestGaugeOperations() {
 		}
 	}
 }
-
-func (suite *KeeperTestSuite) TestChargeFeeIfSufficientFeeDenomBalance() {
```

Comment:
moved to [msg_server_test.go](https://github.com/dymensionxyz/dymension/pull/1113/files#diff-8ca6005e3abcf6a34915541d0ba4c6a1011475ba9c46e58922404bc26451539bR380)